composer.json

```diff
@@ -23,7 +23,7 @@
     "require": {
         "firebase/php-jwt": "^5.5|^6.0",
         "guzzlehttp/guzzle": "^7.0",
-        "phpseclib/phpseclib": "^2.0"
+        "phpseclib/phpseclib": "^3.0"
```

Comment:
`phpseclib` was a major version bump b/c their public interface requires PHP 5.6 instead of 5.3 as the minimum. We're also not exposing any of their internals through our public API (AFAIK), so I think this should just be a minor version bump.